canonical-websites/www.ubuntu.com
# 18.10 webinar created

## Done

Created 18.10 takeover according to [copy doc](https://drive.google.com/file/d/0B5Jc5K2-I_6xS1E4OWMwd0VNSkx5elJJZWpFMkdEUXhNQk5z/view)
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the [design spec](https://github.com/ubuntudesign/design.ubuntu.com-design/issues/15)  and the [copy doc](https://drive.google.com/file/d/0B5Jc5K2-I_6xS1E4OWMwd0VNSkx5elJJZWpFMkdEUXhNQk5z/view) refresh the page until the takeover for 18.10 webinar loads. Make sure that it matches the design documents.


## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4284
## Screenshots

![image](https://user-images.githubusercontent.com/43535482/47665526-3de99e80-db99-11e8-987f-bca699aa9ae6.png)

![image](https://user-images.githubusercontent.com/43535482/47665575-55288c00-db99-11e8-9d40-69dbafca639b.png)
